plugin.yml

```diff
@@ -0,0 +1,25 @@
+name: Smooth Secrets
+description: Plugin to manage your secrets
+author: https://github.com/hasura
+requirements: []
+configuration:
+  properties:
+    secrets:
+      type: array
+      properties:
+        strategy:
+          type: string
+        key:
+          type: string
+        type:
+          type: string
+        region:
+          type: string
+      required:
+        - strategy
+        - key
+        - type
+        - region
```

Comment:
I think we should be using the `items` property here and move this object definition inside that. https://json-schema.org/understanding-json-schema/reference/array.html#id8

(at the same time I wonder why json scehma validation isn't being done on buildkite's side :thinking: )